api-wrapper/services/api-wrapper/src/test/java/net/catenax/edc/apiwrapper/config/ApiWrapperConfigTest.java

```diff
@@ -13,17 +13,18 @@ public class ApiWrapperConfigTest {
     @Test
     void testBuild() {
 
-        Map<String, String> map = new HashMap<>();
-        map.put("userId", "userPwd");
+        List<BasicAuthVaultLabels> credentialsList = new ArrayList<>();
+        BasicAuthVaultLabels credentials = new BasicAuthVaultLabels("UserId","UserPwd");
+        credentialsList.add(credentials);
```

Comment:
Avoid logic, where it's not necessary ;-)

```suggestion
        List<BasicAuthVaultLabels> credentialsList = List.of(
                new BasicAuthVaultLabels("UserId", "UserPwd")
        );
```